src/common/helpers/github-url.js

```diff
@@ -1,6 +1,16 @@
-const getGitHubRepoUrl = (owner, name) => {
+import parseGitHubUrl from 'parse-github-url';
+
+export const getGitHubRepoUrl = (owner, name) => {
   let repository = !name ? owner : `${owner}/${name}`;
   return `https://github.com/${repository}`;
 };
 
-export default getGitHubRepoUrl;
+export const parseGitHubRepoUrl = (url) => {
+  const repo = parseGitHubUrl(url);
+  return (repo && repo.repo) ? {
+    owner: repo.owner,
+    name: repo.name,
+    fullName: repo.repo,
+    url: url
+  } : null;
```

Comment:
Personally I find naming repository full name 'repo' a bit confusing. Especially when nested in some "repository" object, so I prefer to stick to naming used by GH in their API.